src/js/scripts.js

```diff
@@ -6,15 +6,19 @@ if (
   issetCookie('cone-cookie-law-analytics')
   && getCookie('cone-cookie-law-analytics') === 'accepted'
```

Comment:
isset may be unnecessary
https://github.com/conedevelopment/cone-site/blob/66b616bb81ee51ae72559a6be125b26a13e125e9/src/js/cookie.js#L36-L40

Reply:
You are right Viktor! Thanks!